integration-tests/api/__tests__/admin/order.js

```diff
@@ -620,6 +636,47 @@ describe("/admin/orders", () => {
         }),
       ]);
     });
+
+    it("Only allow canceling claim after canceling return and fulfillment", async () => {
```

Comment:
A rather large integration test which uses the newly added utility functions to ease execution.
Firstly, it is attempted to cancel an order, which should be illegal. Hereafter, we cancel its content one type at a time, and try to cancel the order which should still be illegal - until all content (fullfillments, claims, swaps ...) are canceled.